OauthRenew/oauthrenew/_version.py

```diff
@@ -0,0 +1,3 @@
+# please don't modify this file,
+# this is automatically updated by bump2version
+__version__ = '0.0.1'
```

Comment:
Tag the versions as 0.0.0 so that when we bump the version (it's the process that then will build everything automatically), we can have 0.0.1.